src/components/FormElements/ResourceFeePickerWithQuery.tsx

```diff
@@ -103,12 +104,27 @@ export const ResourceFeePickerWithQuery = ({
 
   // Create a sample transaction to simulate to get the min resource fee
   const buildTxToSimulate = () => {
+    setErrorMessage("");
+
     try {
-      const contractDataXDR = getContractDataXDR({
-        contractAddress: operation.params.contract,
-        dataKey: operation.params.key_xdr,
-        durability: operation.params.durability,
-      });
+      let contractDataXDR;
```

Comment:
We prefer using "pure" camelcase for consistency.

```suggestion
      let contractDataXdr;
```